fastly/resource_fastly_alert_test.go

```diff
@@ -193,6 +193,69 @@ func TestAccFastlyAlert_basic_stats_aggregate(t *testing.T) {
 	})
 }
 
+func TestAccFastlyAlert_BasicStatsAggregatePercent(t *testing.T) {
+	service := gofastly.ServiceDetail{
+		Name:      gofastly.ToPointer(""),
+		ServiceID: gofastly.ToPointer(""),
+	}
+
+	createAlert := gofastly.AlertDefinition{
+		Description: "Terraform percent test",
+		Dimensions:  map[string][]string{},
+		// 25 percent increase
+		EvaluationStrategy: map[string]any{
+			"type":         "percent_increase",
```

Comment:
Nice, I see we're adding tests for the new evaluation strategies, but the docs string for `type` doesn't call out these new options.

Reply:
True, I added the missing ones, and the missing evaluation periods.